reference/lexical_analysis.po

```diff
@@ -877,110 +1192,151 @@ msgid ""
+"Une conséquence de partager la même syntaxe avec les chaînes littérales "
+"normales est que les caractères dans les champs à remplacer ne doivent pas "
+"entrer en conflit avec le guillemet utilisé pour encadrer la chaine formatée "
+"littérale ::"
 
 #: ../Doc/reference/lexical_analysis.rst:692
 msgid ""
 "Backslashes are not allowed in format expressions and will raise an error::"
 msgstr ""
+"La barre oblique inversée n'est pas autorisée dans les expressions des "
+"champs à remplacer et son utilisation génère une erreur ::"
 
 #: ../Doc/reference/lexical_analysis.rst:697
 msgid ""
 "To include a value in which a backslash escape is required, create a "
 "temporary variable."
 msgstr ""
+"Pour inclure une valeur où l'échappement par barre oblique inversée est "
+"nécessaire, vous devez créer une variable temporaire."
 
 #: ../Doc/reference/lexical_analysis.rst:704
 msgid ""
 "Formatted string literals cannot be used as docstrings, even if they do not "
 "include expressions."
 msgstr ""
+"Une chaine formatée littérale ne peut pas être utilisée en tant que "
+"docstring, même si elle ne comporte pas d'expression."
 
 #: ../Doc/reference/lexical_analysis.rst:715
 msgid ""
 "See also :pep:`498` for the proposal that added formatted string literals, "
 "and :meth:`str.format`, which uses a related format string mechanism."
 msgstr ""
+"Consultez aussi la :pep:`498` qui propose l'ajout des chaines formatées "
+"littérales et :meth:`str.format` qui utilise un mécanisme similaire pour "
+"formater les chaînes de caractères."
 
 #: ../Doc/reference/lexical_analysis.rst:722
 msgid "Numeric literals"
-msgstr ""
+msgstr "Littéraux numériques"
 
 #: ../Doc/reference/lexical_analysis.rst:728
 msgid ""
 "There are three types of numeric literals: integers, floating point numbers, "
 "and imaginary numbers.  There are no complex literals (complex numbers can "
 "be formed by adding a real number and an imaginary number)."
 msgstr ""
+"Il existe trois types de littéraux numériques : les entiers, les nombres à "
+"virgule flottante et les nombres imaginaires. Il n'y a pas de littéraux "
+"complexes (les nombres complexes peuvent être construits en ajoutant un "
+"nombre réel et un nombre imaginaire)."
 
 #: ../Doc/reference/lexical_analysis.rst:732
 msgid ""
 "Note that numeric literals do not include a sign; a phrase like ``-1`` is "
 "actually an expression composed of the unary operator '``-``' and the "
 "literal ``1``."
 msgstr ""
+"Notez que les littéraux numériques ne comportent pas de signe ; une phrase "
+"telle que ``-1`` est en fait une expression composée de l'opérateur unitaire "
+"'``-``' et du littéral ``1``."
 
 #: ../Doc/reference/lexical_analysis.rst:740
 msgid "Integer literals"
-msgstr ""
+msgstr "Entiers littéraux "
 
 #: ../Doc/reference/lexical_analysis.rst:742
 msgid "Integer literals are described by the following lexical definitions:"
 msgstr ""
+"Les entiers littéraux sont décrits par les définitions lexicales suivantes ::"
```

Comment:
It look like a space, you should use a non-breakable space here (or " : ::").

Reply:
Je corrige l'ensemble des erreurs.

Reply:
J'ai vu ton commit, j'ignore si il signifie "ça y est j'ai corrigé", mais il reste pas mal de " ::". J'ignore si on en avait déjà parlé mais sphinx-doc n'affiche pas de ":" lorsqu'il voit un " ::", il faut donc soit écrire " : ::" (un peu lourd) soit " ::" (avec un espace insécable). (Bon j'espère que c'est pas github qui m'affiche des espaces là où tu met des espaces insécables ?).